recipe/recipe.yaml

```
@@ -0,0 +1,108 @@
+schema_version: 1
+
+context:
+  name: katago
+  version: 1.15.3
+  cuda: ${{ "true" if cuda_compiler_version != "None" else "false" }}
+  cuda_build_string: cuda_${{ cuda_compiler_version | version_to_buildstring }}
+  string_prefix: ${{ cuda_build_string if cuda == "true" else "cpu_" }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  # use git_url, because katago needs a git repo to compile BUILD_DISTRIBUTED support;
+  # this cannot (or rather: should not) be patch out, or even patched at all (-> dirty tag),
+  # compare the exceedingly clear instructions/pleas here:
+  # https://github.com/lightvector/KataGo/blob/master/Compiling.md#building-for-distributed
+  git: https://github.com/lightvector/KataGo.git
+  tag: v${{ version }}
+
+build:
+  number: 1
+  skip:
+    - cuda == "true" and win
+    - cuda == "true" and match(cuda_compiler_version, "<12")
+  string: ${{ string_prefix }}h${{ hash }}_${{ build_number }}
+  variant:
+    use_keys:
+      # use cuda from the variant config, e.g. to build multiple CUDA variants
+      - ${{ "cuda" if cuda == "true" }}
+    # this will down-prioritize the cuda variant versus other variants of the package
+    down_prioritize_variant: ${{ 1 if cuda == "true" else 0 }}
+
+requirements:
+  build:
+    - if: cuda == "true"
+      then:
+        - ${{ compiler('cuda') }}
+        - cuda-version ==${{ cuda_compiler_version }}
+        - libcublas-dev
+
+    - if: build_platform != target_platform
+      then:
+        - git
```

Is this always needed?

```suggestion
    - git
```

---

See comment in main thread about git.